test/quantization/test_quantize.py

```diff
@@ -536,6 +536,24 @@ def test_quantized_embedding(self):
         self.assertTrue('QuantizedLinear' in str(model))
         self.checkQuantizedLinear(model.fc)
 
+    @skipIfNoFBGEMM
+    def test_dequant_stub(self):
+        m = QuantStubModel().eval()
+        prepare(m, inplace=True)
+        self.checkObservers(m)
+        convert(m, inplace=True)
+        self.assertEqual(type(m.quant), nnq.Quantize)
+        self.assertEqual(type(m.dequant), nnq.DeQuantize)
+
+        # check DeQuantStub is not swapped when it doesn't have a qconfig
+        m = QuantStubModel().eval()
+        m.dequant.qconfig = None
+        prepare(m, inplace=True)
+        self.checkObservers(m)
+        convert(m, inplace=True)
+        self.assertEqual(type(m.quant), nnq.Quantize)
+        self.assertEqual(type(m.dequant), DeQuantStub)
```

Comment:
optional nit: not shadowing `m` (i.e. use `m2` or something similar) would reduce the risk of someone changing the code incorrectly in the future

Reply:
sure